hw/mcu/stm/stm32_common/src/stm32_driver_mod_i2c_v1.c

```diff
@@ -373,7 +373,18 @@ HAL_StatusTypeDef HAL_I2C_Master_Transmit_Custom(I2C_HandleTypeDef *hi2c,
       /* Wait until BUSY flag is reset */
       if(I2C_WaitOnFlagUntilTimeout(hi2c, I2C_FLAG_BUSY, SET, I2C_TIMEOUT_BUSY_FLAG, tickstart) != HAL_OK)
       {
-        return HAL_BUSY;

```

Comment:
I wonder if this needs to be applied to other families as well or just F1?

Reply:
Same suggestion in all datasheet for F1/F4/L1, commit message changed.